packages/chat-component/src/styles/chat-thread-component.ts

```diff
@@ -12,13 +12,22 @@ export const styles = css`
     }
     100% {
       opacity: 1;
-      top: 0px;
+      top: 0;
     }
   }
+  .chat__header--button,
+  .chat__header--avatar {
+    display: flex;
+    align-items: center;
+  }
```

Comment:
I think this no longer is in the thread component no?

Reply:
removed.